pkg/health/health_test.go

```diff
@@ -39,52 +39,52 @@ func TestHealth(t *testing.T) {
 var _ = Describe("health", func() {
 	Context("CheckCustomResourceDefinition", func() {
 		DescribeTable("crds",
-			func(crd *apiextensionsv1beta1.CustomResourceDefinition, matcher types.GomegaMatcher) {
+			func(crd *apiextensionsv1.CustomResourceDefinition, matcher types.GomegaMatcher) {
```

Comment:
Can you cover both API versions in the test here? I.e. make sure, that it still checks the same stuff for the old API version?

Reply:
Sorry, you're right.